src/maker/config.rs

```diff
@@ -143,6 +151,27 @@ impl MakerConfig {
     }
 }
 
+fn create_default_maker_dirs(default_maker_config_path: &PathBuf) {
```

Comment:
A more clearer name for this could be

```suggestion
fn create_default_maker_config(target_path: &PathBuf) {
```